tests/integration/data/lint.json

```diff
@@ -27,8 +27,8 @@
         {
             "name": "lint group",
             "tasks": [
-                "src",
-                "test"
+                "make-lint-src",
```

Comment:
The changes in this file and the next file were actually a bug in the old test code. The way v2 is designed, this bug can't occur.